db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.3.0_upgrade.sql

```diff
@@ -0,0 +1,20 @@
+-- ------------------------------------------------------------------------------------------
+-- Upgrade script for Migrating Spring batch version to 5.0 as part of Java 21 Migration.
+-- ------------------------------------------------------------------------------------------
+
+ALTER TABLE  master.BATCH_STEP_EXECUTION ADD CREATE_TIME TIMESTAMP NOT NULL DEFAULT '1970-01-01 00:00:00';
+ALTER TABLE  master.BATCH_STEP_EXECUTION ALTER COLUMN START_TIME DROP NOT NULL;
```

Comment:
This alter statement is the same the one in the rollback script!

Reply:
The **ALTER TABLE master.BATCH_STEP_EXECUTION ALTER COLUMN START_TIME DROP NOT NULL**; statement appears in both the upgrade and rollback scripts because it is the same operation in both cases: removing the NOT NULL constraint from the START_TIME column.